x-pack/plugins/security_solution/public/types.ts

```diff
@@ -49,6 +50,7 @@ export interface SetupPlugins {
 export interface StartPlugins {
   cases: CasesUiStart;
   data: DataPublicPluginStart;
+  dashboard: DashboardStart;
```

Comment:
We've declared `dashboard` as an optional dependency, but this is saying it will be always available.
```suggestion
  dashboard?: DashboardStart;
```

We should consider what to do in the case where this plugin is disabled, but I assume it's the same as if dashboards are not installed.